test/parallel/test-http-client-headers-host-array.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
+const http = require('http');
+
+{
+
+  const options = {
+    port: '80',
+    path: '/',
+    headers: {
+      host: []
+    }
+  };
+
+  assert.throws(() => {
+    http.request(options);
+  }, {
+    code: /ERR_HTTP_INVALID_HEADER_VALUE/,
+    message: 'Invalid value "[]" for header "host"'
```

Comment:
If switching to `ERR_INVALID_ARG_TYPE`, I would just omit the message check here.
```suggestion
```

Reply:
Removed the message.